### PR TITLE
fix(#1934): make displaying own DRep in DRep directory available

### PR DESCRIPTION
**Pull request by MSzalowski**
## List of changes

- make displaying own DRep in DRep directory available

## Checklist

- [related issue](https://github.com/IntersectMBO/govtool/issues/1934)
- [x] My changes generate no new warnings
- [x] My code follows the [style guidelines](https://github.com/IntersectMBO/govtool/tree/main/docs/style-guides) of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the [changelog](https://github.com/IntersectMBO/govtool/blob/main/CHANGELOG.md)
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by MSzalowski** [comment]
@j-dyczka Once accepted - could you merge it?
Also, if any changes are needed, could you handle them? I will be off for some time, so to not block this PR, please do whatever might be needed to make it done :raised_hands:

File: govtool/frontend/src/pages/DRepDirectoryContent.tsx
Comment (jdyczka) [inline govtool/frontend/src/pages/DRepDirectoryContent.tsx:214]: Shouldn't we still exclude delegation in progress since it's displayed at the top of the page?

Comment (jdyczka) [reply govtool/frontend/src/pages/DRepDirectoryContent.tsx:214]: I was wrong, it's not

**Review by jdyczka — APPROVED** [review]

**Pull request merged**
